### PR TITLE
ublox allow setting dynModel

### PR DESCRIPTION
**Pull request by dagar**
https://github.com/PX4/Firmware/pull/7470

### CODE REVIEW

File: src/ubx.h [hunk 1]
```diff
@@ -671,6 +673,9 @@ class GPSDriverUBX : public GPSHelper
 	const Interface		_interface;
 	uint32_t _survey_in_acc_limit;
 	uint32_t _survey_in_min_dur;
+
+	// ublox Dynamic platform model default 7: airborne with <2g acceleration
+	uint8_t _dynModel{7};
```
Comment (bkueng) [inline src/ubx.h]: nitpick: naming convention: `_dyn_model`

File: src/ubx.h [hunk 2]
```diff
@@ -571,13 +569,17 @@ class GPSDriverUBX : public GPSHelper
 public:
 	GPSDriverUBX(Interface gpsInterface, GPSCallbackPtr callback, void *callback_user,
 		     struct vehicle_gps_position_s *gps_position,
-		     struct satellite_info_s *satellite_info);
+		     struct satellite_info_s *satellite_info,
+		     uint8_t dynamic_model);
```
Comment (bkueng) [inline src/ubx.h]: add default argument `dynamic_model=7`, so that updating the submodule in QGC will just work?

Comment (dagar) [inline src/ubx.h]: I actually did that initially, but assumed QGC was the only one setting it to 2?

File: src/ubx.h (hunk 1 above)
Comment (dagar) [inline src/ubx.h]: It was to match the field straight out of ublox documentation, but I'll change it.

File: src/ubx.h (hunk 2 above)
Comment (bkueng) [inline src/ubx.h]: I thought because you have `RTCM mode force stationary dynamic model`, QGC gets it right anyway. I'm ok with either way.

Comment (dagar) [inline src/ubx.h]: Right, thanks.

**Comment by dagar** [comment]
Updated

**Pull request merged**